src/components/collapsable-box/collapsable-box.test.js

```diff
@@ -2,7 +2,14 @@ import { shallow } from 'enzyme'
 import CollapsableBox from './collapsable-box'
 
 describe('CollapsableBox ', () => {
-  test('clicking the header changes the expanded state', () => {
+  test('clicking the header changes the expanded state on mobile/tablet', () => {
```

Comment:
I think it would be worthwhile to write a test that shows that all of the boxes are open by default, without passing `expanded=false` as an attribute

Reply:
done